src/app/settings/stix-objects/course-of-actions/course-of-action/course-of-action.component.spec.ts

```diff
@@ -1,31 +1,53 @@
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { inject, async, TestBed,  ComponentFixture } from '@angular/core/testing';
+import { Location } from '@angular/common';
+import { MdSnackBar, MdDialog } from '@angular/material';
+import { Router, ActivatedRoute } from '@angular/router';
 
 // Load the implementations that should be tested
 import { CourseOfActionComponent } from './course-of-action.component';
+<<<<<<< HEAD
```

Comment:
Can you resolve these commit merge markers?

Reply:
This should be fix